frontend/app/src/App.js

```diff
@@ -1,25 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+
+import TextField from '@mui/material/TextField';
+
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+
+const Input = createBrowserRouter([
+  {
+    path: "/documents/new",
+    element: <div>
+      <TextField id="outlined-basic" label="Outlined" variant="outlined" />
+    </div>,
+  },
+]);
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <RouterProvider router={Input} />
   );
 }
 
-export default App;
+export default App;
```

Comment:
There should always be a newline in the end of the line, keep the consistency please )